ops/setup-environment.sh

```diff
@@ -73,7 +73,7 @@ cd ../fortis-interface || exit 2
 
 {
 echo APPINSIGHTS_INSTRUMENTATIONKEY="${app_insights_id}"
-echo FORTIS_SERVICE_HOST="${graphql_service_host}"
+echo REACT_APP_SERVICE_HOST="${graphql_service_host}"
```

Comment:
The reason why this env var exists is because it's used by create-react-app which is dependent on this naming convention.